GSA-TTS/FAC
# 4385 remove finding idyn uniqueness validation

## Description
This PR is to turn off finding uniqueness check. See #4385 for more details.

## How to Test:
1. Launch the app locally from the main branch.
2. Try uploading a findings workbook that contains repeated findings, and note the error message (refer to the example in the attachment).
3. Switch to the PR branch and upload the same workbook. There should be no error.
[federal-awards-audit-findings-workbook.xlsx](https://github.com/user-attachments/files/17397111/federal-awards-audit-findings-workbook.xlsx)


## PR Checklist: Submitter


## PR Checklist: Reviewer


The larger the PR, the stricter we should be about these points.

## Pre Merge Checklist: Merger
